wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -1680,12 +1681,9 @@ public final class MLSService: MLSServiceInterface {
                     "Handling reset broken MLS conversation recovery strategy...",
                     attributes: [.mlsGroupID: groupID.safeForLoggingDescription]
                 )
-                var epoch: Int64 = 0
-                if let context, let conversation = fetchConversationInfo(
-                    with: groupID,
-                    in: context
-                )?.conversation {
-                    epoch = Int64(conversation.epoch)
```

Comment:
this part getting the epoch outside the CoreData context was failing on debug build